packages/rrweb-snapshot/test/html/video.html

```diff
@@ -7,7 +7,7 @@
     <title>video</title>
   </head>
   <body>
-    <video controls>
```

Comment:
Should we expect a snapshot file diff?

Reply:
No, because we are removing the autoplay attribute :D the snapshot changes without the change (which is expected), so IMHO the fact that this stays static is basically the test for this.

Reply:
oh right :)